persian-calendar@iamrezamousavi.gmail.com/hijriDate.js

```diff
@@ -139,8 +139,8 @@ function _hilalIM(country = 'IR') {
             startYear: 1427, /* =iDoM:firstYear */
             startJD: 2453767, /* =_hijriAToJulianDay(startYear,1,1) */
 
-            endYear: 1447, /* =iDoM:lastYear */
-            endJD: 2461207, /* =_hijriAToJulianDay(endYear,endMonth+1,1)-1 */
+            endYear: 1448, /* =iDoM:lastYear */
+            endJD: 2461120, /* =_hijriAToJulianDay(endYear,endMonth+1,1)-1 */
```

Comment:
Its not true. to fix this follow the step below:
1. Install `nodejs`
2. add `console.log(_hijriAToJulianDay(1448,9+1,1)-1)` to end of hijriDate.js
3. run `node hijriDate.js`

Reply:
Fixed, thanks

Reply:
I'm terribly sorry, I did change that to the number but didn't save the file before the amend… 😅 Fixed. Thanks

Reply:
Sorry again, done